Oracle/PCORNetLoader_ora.sql

```diff
@@ -1471,7 +1471,7 @@ insert into prescribing (
 --    ,RAW_RXNORM_CUI
 )
 select distinct  m.patient_num, m.Encounter_Num,m.provider_id,  m.start_date order_date,  to_char(m.start_date,'HH:MI'), m.start_date start_date, m.end_date, mo.pcori_cui
-    ,quantity.nval_num quantity, refills.nval_num refills, supply.nval_num supply, freq.pcori_basecode frequency, 
+    ,quantity.nval_num quantity, refills.nval_num refills, supply.nval_num supply, substr(freq.pcori_basecode, instr(freq.pcori_basecode, ':') + 1, 2) frequency, 

```

Comment:
@dckc, it's because the SCILHS medication ontology has things like `RX_FREQUENCY:02` in the pcori_basecode column.  See [pcornet_med.zip](https://github.com/SCILHS/scilhs-ontology/blob/master/Ontology/pcornet_med.zip)